rotkehlchen/chain/substrate/typing.py

```diff
@@ -40,15 +41,52 @@ def endpoint(self) -> str:
                 'via "own_rpc_endpoint" or the specific db setting',
             )
         if self == KusamaNodeName.PARITY:
-            return 'https://kusama-rpc.polkadot.io/'
```

Comment:
Why this change?

Reply:
changed it for testing and forgot to change back